src/inspected-window-helpers/overlay-helpers.js

```diff
@@ -33,11 +33,9 @@ export function setupOverlayHelpers() {
 
   function getOverlayNodesAndOptions(app) {
     const { selectors, options = {} } = app.devtools.overlays;
-    if (!selectors.length) {
-      selectors.push(`#single-spa-application\\:${app.name}`);
-    }
     return {
       nodes: selectors
+        .concat(`#single-spa-application\\:${app.name}`)
```

Comment:
Is `selectors` always defined, even for apps that don't export `app.devtools.overlays`?

Reply:
I thought so but I'm failing to find where that is (I think it was in either the main single-spa or the helpers, but I thought they merged in a default set of options, one of them being assigning selectors to an empty array). Shouldn't be a problem to also use a default assignment here.

Reply:
I think here is where I'm making that assumption from: https://github.com/CanopyTax/single-spa/blob/983df96cd0a8d9019a241dbd81702d6518b72daa/src/applications/apps.js#L65\

Still, no real reason not to include the default assignment in here. Will add.